content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst

```diff
@@ -4,128 +4,141 @@ Process receipts and deliveries in one step
 
 .. _inventory/receipts_delivery_one_step:
 
-By default, incoming shipments are configured to be received directly into stock, and outgoing
-shipments are configured to be delivered directly from stock to the customer; the default setting
-for warehouses in Odoo is one step receipts and deliveries.
+.. |PO| replace:: :abbr:`PO (Purchase Order)`
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+.. |RfQ| replace:: :abbr:`RfQ (Request for Quotation)`
 
-.. note::
-   Incoming and outgoing shipments do not need to be configured with the same steps. For example,
-   products can be received in one step, but shipped in three steps.
+In Odoo *Inventory*, both incoming and outgoing shipments are configured to process in one step, by
+default. This means purchases will be received directly into stock, and deliveries will be moved
+directly from stock to customers.
 
-In the following example, one step will be used for both receipts and deliveries.
+.. tip::
+   Incoming and outgoing shipments do **not** need to be configured with the same amount of steps.
 
-.. _inventory/receipts_delivery_one_step/wh:
+   For example, a warehouse's settings can be configured so products can be received directly in one
+   step, and delivered in three steps (pick + pack + ship).
 
-Configure the warehouse
-=======================
+Configuration
+=============
 
-If another receiving or shipping configuration is set on the warehouse, they can easily be set back
-to the one step setting.
+To configure one-step receipts and deliveries for a warehouse, navigate to :menuselection:`Inventory
+app --> Configuration --> Warehouses`, and select a warehouse to edit.
 
-Begin by navigating to :menuselection:`Inventory --> Configuration --> Warehouses`, and click on the
-desired warehouse to edit. Next, on the :guilabel:`Warehouse Configuration` tab, under the
-:guilabel:`Shipments` section, select :guilabel:`Receive goods directly (1 step)` for
-:guilabel:`Incoming Shipments` and/or :guilabel:`Deliver goods directly (1 step)` for
-:guilabel:`Outgoing Shipments`.
+Under the :guilabel:`Warehouse Configuration` tab, set :guilabel:`Incoming Shipments` to
+:guilabel:`Receive goods directly (1 step)`, and set :guilabel:`Outgoing Shipments` to
+:guilabel:`Deliver goods directly (1 step)`.
 
-.. image:: receipts_delivery_one_step/one-step-warehouse-config.png
+.. image:: receipts_delivery_one_step/receipts-delivery-one-step-warehouse-settings.png
    :align: center
-   :alt: Set incoming and outgoing shipment options to receive and deliver in one step.
+   :alt: Incoming and outgoing shipments set to one-step on warehouse form.
+
+.. note::
+   Since one-step receipt and delivery is the default for incoming and outgoing shipments in Odoo,
+   the *Multi-Step Routes* feature is *not* required.
+
+   However, for the above settings to appear on a warehouse form, the feature **must** be enabled.
+
+   To enable *Multi-Step Routes*, navigate to :menuselection:`Inventory app --> Configuration -->
```

Comment:
I am not sure if this is necessary in this doc? Or instea you can link to the other docs- the 2 and 3 step docs?

Reply:
I understand the logic here, but because I show the shipments settings in the included screenshot, I think it's helpful to mention that this is required for that to show up. But again, if I get similar feedback, I'll adjust accordingly!